gulpfile.js

```diff
@@ -1,39 +1,59 @@
 'use strict';
 
 var gulp = require('gulp');
+var clangFormat = require('clang-format');
+var gulpFormat = require('gulp-clang-format');
 var runSequence = require('run-sequence');
-var spawnSync = require('child_process').spawnSync;
+var spawn = require('child_process').spawn;
+
+var runSpawn = function(done, task, opt_arg) {
+  var child = spawn(task, opt_arg);
+  child.stdout.on('data', function(data) {
```

Comment:
Instead of manually using console.log, can we use child_process stdio options? (https://nodejs.org/dist/latest-v4.x/docs/api/child_process.html#child_process_options_stdio) I think 'inherit' does what you want here.